### PR TITLE
Moved more js to own files

### PR DESCRIPTION
**Pull request by OmgImAlexis**

### CODE REVIEW

File: gui/slick/views/displayShow.mako [hunk 1]
```diff
@@ -540,8 +493,13 @@ $(document).ready(function(){
                     ${file_size}
                 % endif
             </td>
+            ## <% date = (sbdatetime.sbdatetime.sbfdate(sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network))), 'never')[int(epResult['airdate']) == 1] %>
+            <% date = sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network)) %>
```
Comment (OmgImAlexis) [inline gui/slick/views/displayShow.mako]: @MGaetan89 any chance we could get some date/time formatting functions added in python so we don't have to do this every time?

Comment (miigotu) [inline gui/slick/views/displayShow.mako]: Why did you remove the "never" cases?

Comment (OmgImAlexis) [inline gui/slick/views/displayShow.mako]: Oops, I thought I had added that back in. Give me a sec.

File: gui/slick/js/new/config_postProcessing.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$('#config-components').tabs();
```
Comment (MGaetan89) [inline gui/slick/js/new/config_postProcessing.js]: I think we can get rid of all the `.tabs()` calls by using `data-toggle="tab"`.
http://getbootstrap.com/javascript/#tabs

File: gui/slick/js/new/config_subtitles.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
```
Comment (MGaetan89) [inline gui/slick/js/new/config_subtitles.js]: We do you have this only here (and still some JS outside a few lines below)?

File: gui/slick/views/displayShow.mako (hunk 1 above)
Comment (MGaetan89) [inline gui/slick/views/displayShow.mako]: Yes, I agree about having a common function to format the date.
I'm always finding something else to do on SickRage, so it's hard to have a prioritized list.

File: gui/slick/js/new/config_subtitles.js (hunk 3 above)
Comment (OmgImAlexis) [inline gui/slick/js/new/config_subtitles.js]: For now it's just what was in the mako files, I'm slowly removing this. I'll redo the name and stuff for this PR in a bit.

File: gui/slick/js/new/config_postProcessing.js (hunk 2 above)
Comment (OmgImAlexis) [inline gui/slick/js/new/config_postProcessing.js]: Sounds good.

Comment (OmgImAlexis) [inline gui/slick/js/new/config_postProcessing.js]: Just checked and I think we still need it but I can move them all to a single file.
